packages/kbn-unified-field-list/src/hooks/use_existing_fields.ts

```diff
@@ -178,6 +179,8 @@ export const useExistingFieldsFetcher = (
   const dataViewsHash = getDataViewsHash(params.dataViews);
   const refetchFieldsExistenceInfo = useCallback(
     async (dataViewId?: string) => {
+      const startTime = window.performance.now();
```

Comment:
@jughosta does the way we're thinking of measuring field list loading make sense to you?

Reply:
@drewdaemon Yes, currently we get fields from dataView class and make this additional call to separate "Empty" fields in the sidebar. As long as we specify that it's `fetchFieldsExistenceInfo` request metrics, it's correct.

Reply:
I think what we're really going for the total time it takes to load the field list in Lens.

It sounds like you're saying we can assume that the fields are already present on the client so this existence check is the only network call we need to account for. Is that right?

Reply:
@drewdaemon Yes, right now it's the only request we make for the field list. But it will change in the future. @mattkime is working on RFC to load fields async and separately from data view info. Once we apply this change, we can sum both loading durations.